### PR TITLE
Exercise 3 - Working with Numbers and Math

### PR DESCRIPTION
**Pull request by coverdriver27**
Working with Numbers and Math Operations

### CODE REVIEW

**Review by tony-resendiz — DISMISSED** [review]

**Review by defrank — CHANGES_REQUESTED** [review]
Please remove the file, `Ex04/ex4.py`, and ensure other PRs do not contain files for exercises that are not part of the PR -- even if the files are empty.

File: Ex03/ex3.py [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+print ("Oh, that's why it's False.")
+
+print ("How about some more.")
+
+print ("Is it greater?", 5 > -2) # returns a true or false (Is 5 Greater than -2?)
+print ("Is it greater or equal?", 5 >= -2) # returns a true or false
+print ("Is it less or equal?", 5 <= -2) # return a true or false
+print ("----------------------------------------------------------------")
+print ("Now lets calculate you're annual salary:")
```
[the remark this answers is not in the document]
Comment (coverdriver27) [reply Ex03/ex3.py]: Fixed the grammatical issue.

[the remark this answers is not in the document]
Comment (coverdriver27) [reply Ex03/ex3.py]: Fixed the formatting

File: Ex03/ex3.py [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+print ("Roosters", 100 - 25 * 3 % 4) # % = Modulo, Remainder when a is divided by b
+
+print ("Now i will count the eggs:")
+
+print(3 + 2 + 1 - 5 + 4 % 2 - 1 / 4 + 6)
+
+
+print ("Is it true that 3 + 2 < 5 -7?")
+
+print (3 + 2 < 5 - 7) # returns a true or false
```
[the remark this answers is not in the document]
Comment (coverdriver27) [reply Ex03/ex3.py]: Fixed the Comments

**Review by defrank — APPROVED** [review]
LGTM

**Pull request merged**